src/main/resources/public/js/activity.service.js

```diff
@@ -69,30 +69,32 @@
 
     function enrichWithStatistics(activities) {
       logger.debug('enrichWithStatistics');
-      angular.forEach(activities, function(activity) {
-        var distanceRound = Math.round(activity.distance);
-        var distanceStep = Math.ceil(Math.sqrt(distanceRound));
+      angular.forEach(activities, (activity) => {
+        const distanceRound = Math.round(activity.distance);
+        const distanceStep = Math.ceil(Math.sqrt(distanceRound));
         rateByDistance(activity, activities, distanceRound - distanceStep, distanceRound + distanceStep);
       });
     }
 
     function rateByDistance(activity, activities, minDistance, maxDistance) {
-      var matches = activities.filter(function(current) {
+      const matches = activities.filter((current) => {
         return current.distance > minDistance && current.distance <= maxDistance;
-      }).sort(function(o1, o2) {
+      }).sort((o1, o2) => {
         return o1.timePerKmInSeconds - o2.timePerKmInSeconds;
       });
-      var i = 0;
+      let i = 0;
       while (i < matches.length) {
-        if (activity.timePerKmInSeconds <= matches[i++].timePerKmInSeconds) {
+        if (activity.timePerKmInSeconds <= matches[i].timePerKmInSeconds) {
           break;
         }
+        i += 1;
       }
-      activity.intervalStats = 'Platz ' + i + ' von ' + matches.length + ' zwischen ' + minDistance + ' und ' + maxDistance + 'km';
```

Comment:
This is another (correct) formatting change. According to [Wikipedia](https://de.wikipedia.org/wiki/Wikipedia:Schreibweise_von_Zahlen) the following applies: "Zwischen eine Zahl und die zugehörige Maßeinheit gehört immer ein Leerraum"

example `10 km` instead of `10km`